examples/gaia-fast-list/index.html

```diff
@@ -15,18 +23,9 @@
             <h3>${title}</h3>
             <p>${metadata.body}</p>
           </div>
-          <div class="image round">
-            <img src="${image}"/>
-          </div>
+          <div class="image round"></div>
         </li>
       </template>
     </gaia-fast-list>
-
-    <script src="bower_components/dom-scheduler/lib/dom-scheduler.js"></script>
-    <script src="bower_components/gaia-component/gaia-component.js"></script>
-    <script src="bower_components/poplar/poplar.js"></script>
-    <script src="fast-list.js"></script>
-    <script src="gaia-fast-list.js"></script>
-    <script src="app.js"></script>
```

Comment:
can you backport the manifest changes from #46 (adding the new component permission) to this PR so we get them landed fast?